uber/site_sections/email_admin.py

```diff
@@ -69,13 +68,20 @@ def pending_examples(self, session, ident):
                 )
                 examples.append((model_instance, example))
                 example_count = len(examples)
-                if example_count > 1 or (example_count == 1 and offset > 0):
+                if example_count > 10:
                     break
-            offset += limit
         return {
             'email': email,
             'examples': examples,
+            'message': message,
         }
+    
+    def update_dates(self, session, ident, **params):
```

Comment:
I've always felt that there should be a cool portmanteau of "update date", but I've never been able to make it work.